category/basic.lean

```diff
@@ -20,12 +20,26 @@ run_cmd mk_simp_attr `functor_norm
   g <$> (m <$> x) = (g ∘ m) <$> x :=
 (comp_map _ _ _).symm
 
-@[simp] theorem id_map' (x : f α) : (λa, a) <$> x = x := id_map _
+-- @[simp] theorem id_map' (x : f α) : (λa, a) <$> x = x := id_map _
 
 end functor
 
 section applicative
-variables {f : Type u → Type v} [applicative f] [is_lawful_applicative f]
+variables {f : Type u → Type v} [applicative f]
+
+def mmap₂
```

Comment:
This is now duplicated. It is already in `category` (and I will rename it to `mzip_with`).

Reply:
I do not see it anywhere. Can you double check?